group_project_v2/project_api/api_implementation.py

```diff
@@ -38,10 +38,10 @@ def __init__(self, address, dry_run=False):
         self.dry_run = dry_run
 
     def build_url(self, url_parts, query_params=None, no_trailing_slash=False):
-        url_template = "{}/" + "{}/" * len(url_parts)
-        url_parameters = [self._api_server_address]
-        url_parameters.extend(url_parts)
-        url = url_template.format(*url_parameters)  # pylint: disable=star-args
+        url_template = "{}/" * len(url_parts)
+        url = url_template.format(*url_parts)  # pylint: disable=star-args
```

Comment:
@ziafazal nit: this is a bit messy (but my version was messy too). Do you think something like this would look better:

    url = "/".join(url_parts)
    if not is_absolute(url):  # no change here
        self._api_server_address + "/" + url
    if not no_trailing_slash:
        url+="/"